src/Build/Resources/Strings.resx

```diff
@@ -424,7 +424,7 @@
     <comment>{StrBegin="MSB4101: "}</comment>
   </data>
   <data name="ImportedProjectNotFound" UESanitized="false" Visibility="Public">
-    <value>MSB4019: The imported project "{0}" was not found. Confirm that the path in the &lt;Import&gt; declaration is correct, and that the file exists on disk.</value>
+    <value>MSB4019: The imported project "{0}" was not found. Confirm that the path in the Import declaration "{1}" is correct, and that the file exists on disk.</value>
```

Comment:
Can I ask why you removed the angle brackets around `Import`? I don't necessarily disagree, just curious.

Reply:
Oh yeah, it's just because when we were talking about it, there were no brackets in the changed version you suggested. I just wanted to make sure it was *right*. If you want I can put them back.